### ISSUE TITLE
List of pending-only tweets

### ISSUE THREAD

**Issue by marzman95** [FirstLegoLeague/twitter-admin#8]
Instead of showing only the oldest pending tweet on the homepage, show the (whole) list of pending tweets. This can also be moved to a separate page and that the homepage only shows (more) statistics.

**Comment by poelstra** [comment]
Good idea. Would require a 'proper web app' (instead of these 'plain HTML pages').
Otherwise, refreshing the page might interfere with clicking the Approve/Disapprove buttons.
